src/Fabulous.XamarinForms/Views/Controls/WebView.fs

```diff
@@ -0,0 +1,96 @@
+namespace Fabulous.XamarinForms
+
+open System
+open System.Net
+open System.Runtime.CompilerServices
+open Fabulous
+open Fabulous.XamarinForms
+open Xamarin.Forms
+
+type IWebView =
+    inherit IView
+
+module WebView =
+
+    let WidgetKey = Widgets.register<WebView> ()
+
+    let CanGoBack =
+        Attributes.defineBindable<bool> WebView.CanGoBackProperty
+
+    let CanGoForward =
+        Attributes.defineBindable<bool> WebView.CanGoForwardProperty
+
+    let Source =
+        Attributes.defineBindable<WebViewSource> WebView.SourceProperty
+
+    let Cookies =
+        Attributes.defineBindable<CookieContainer> WebView.CookiesProperty
+
+    let Navigating =
+        Attributes.defineEvent<WebNavigatingEventArgs>
+            "WebView_Navigating"
+            (fun target -> (target :?> WebView).Navigating)
+
+    let Navigated =
+        Attributes.defineEvent<WebNavigatedEventArgs> "WebView_Navigated" (fun target -> (target :?> WebView).Navigated)
+
+    let ReloadRequested =
+        Attributes.defineEventNoArg "WebView_ReloadRequested" (fun target -> (target :?> WebView).ReloadRequested)
+
+[<AutoOpen>]
+module WebViewBuilders =
+    type Fabulous.XamarinForms.View with
+        static member inline private WebView<'msg>(source: WebViewSource) =
```

Comment:
To make more user friendly so they do not need to type WebViewSource.op_Implicit ...  They just need to pass either a string or Uri .

Reply:
Given the class in abstract, it means it can be implemented in different types coming from 3rd party libs.
It doesn't cost us anything to make it public

Reply:
That isa  good point. I have made it public